tests/manage/monitoring/prometheusmetrics/test_monitoring_defaults.py

```diff
@@ -0,0 +1,492 @@
+# -*- coding: utf8 -*-
+"""
+Test cases here performs Prometheus queries directly without a workload, to
+check that OCS Monitoring is configured and available as expected.
+"""
+
+import logging
+import time
+
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import tier1
+from ocs_ci.ocs import constants, defaults, ocp
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.prometheus import PrometheusAPI
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.post_deployment
```

Comment:
This has to be defined in pytest.ini otherwise it will throw warning and most likely fail the execution of pytest.

Reply:
It produces the warning, but it doesn't fail the test execution. That said, the warning is most visible when a test run fails. I discussed this with Daniel, who is working on introducing this marker in the CI runner.

Reply:
Since Daniel corrected me that he won't be changing anything in ocs-ci itself for the CI work I mention above, I'm adding the marker into pytest config file here: https://github.com/red-hat-storage/ocs-ci/pull/1351